openverse_catalog/dags/common/licenses/licenses.py

```diff
@@ -226,6 +226,7 @@ def _build_license_url(license_path) -> str:
     return rewritten_license_url
 
 
+@lru_cache(maxsize=1024)
```

Comment:
Newer versions of Python have `@cache` without an explicit max size. I find that more readable.

Reply:
> _New in version 3.9._

Hell yea! TIL! Thanks Dhruv, changing that now!